src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -881,5 +884,38 @@ object Helpers extends SparkSessionWrapper {
 
   }
 
+  def pipReport(etlDB: String = "overwatch_etl", detailed: Boolean = false):Dataset[Row] = {
```

Comment:
both signatures should return DataFrame not Dataset[Row] (I know they're basically the same thing but this will get rid of the import and conform to the rest of the codebase.